scripts/source-maintenance.sh

```diff
@@ -721,13 +723,31 @@ collectAuthors ()
         return 0 # successfully did nothing, as requested
     fi
 
-    vettedCommitPhraseRegex='[Rr]eference point for automated CONTRIBUTORS updates'
+    vettedPhraseRegex='[Rr]eference point for automated CONTRIBUTORS updates'
+
+    vettedByPr=false
+    if test -n "${PULL_REQUEST_NUMBER}"
+    then
+        vettedByPr=`gh pr view $PULL_REQUEST_NUMBER --json body --jq ".body | test(\"${vettedPhraseRegex}\")"`
+        ghResult=$?
+        if test $ghResult -ne 0
+        then
+            echo "ERROR: Unable to determine whether the PR description contains the vetting phrase"
+            return 1
```

Comment:
To better justify the introduction of ghResult variable and to follow a common `foo | return` pattern that returns the exit code of a failed command:

```suggestion
            return $ghResult
```

Reply:
Done.